README.md

```diff
@@ -315,3 +316,63 @@ Comp 40 gives you the tools to explore a large number of other topics:
 * Mark Sheldon gets very excited about some of the topics in Comp 40. Stop by his office to chat with him!
 * Norman Ramsey designed the class
 * Noah Mendelsohn is the [chairman of the internet](http://www.arcanedomain.com/computerwork.html) and is in general incredibly smart; he teaches COMP 40 on alternating semesters with Prof. Sheldon. Talk to him too!
+
+## COMP 15
+
+### Topics Covered:
+* Your first guide to getting a software engineering job (interview questions!)
+* Knowing the difference between the client side and the implementation side 
+* Introduction to C++ programs that utilize complex data structures
+* Utilizing Object Oriented Programming (OOP) concepts to write better code
+* Using abstraction/modularity to break down complex problems
+* Allocating and managing dynamic memory and visualizing what happens under the hood (aliasing, deep vs shallow copies)
+* Evaluating different data structures (sequences vs linked list, stacks vs queues) 
+* Manipulating data structures to make it better (trees, heaps and more!)
+* Writing code that must handle MANY test cases and handling exceptions
+* Understanding Big O (asymptotic analysis) and why certain choices are better (first step towards becoming an ‘engineer’)
+* Reading long documentations (~8 pages PDF) for homework assignments
+* Writing technical comments and documents (README) that make your code easier to follow
+* RECURSION, recursion, ………, Mark loves RECURSION, hence, so should you
```

Comment:
This advice should be agnostic of the professor that is currently teaching the class. I would avoid mentioning Mark specifically.  And recursion has obvious merits besides just that the professor loves it.